Cross.toml

```diff
@@ -0,0 +1,5 @@
+[build]
+pre-build = [
+    "dpkg --add-architecture $CROSS_DEB_ARCH",
+    "apt-get update && apt-get install --assume-yes libssl-dev:$CROSS_DEB_ARCH libpq-dev:$CROSS_DEB_ARCH"
```

Comment:
Just curious, does it mean that cross compilation doesn't work on Windows/macOS?

Reply:
`cross` creates a Docker container behind the scenes, so, _I think_, it could work.